DESCRIPTION

```diff
@@ -23,14 +23,15 @@ Imports:
     checkmate,
     rlang,
     dplyr,
-    arrow,
     lobstr,
     magrittr
-Suggests: 
+Suggests:
+    arrow,
     testthat (>= 3.0.0),
     spelling
 Config/testthat/edition: 3
 Depends: 
     R (>= 2.10)
 LazyData: true
 Language: en-US
+Additional_repositories: https://p3m.dev/cran/2024-02-02/
```

Comment:
CRAN acknowledges this field but doesn't use it. `install.packages()` does not use it. `remotes::install_*()` *does* use it, but `pak::pak()` does not (yet) :shrug:

This PPPM repo is pinned to the current arrow release version. Alternatively, you can instead get the latest pre-release version of arrow by instead using `Additional_repositories: https://apache.r-universe.dev/` or `Additional_repositories: https://nightlies.apache.org/arrow/r`.